kokoro/config/test/third_party_apps/release.gcl

```diff
@@ -0,0 +1,30 @@
+import 'common.gcl' as common
+import '../image_lists.gcl' as image_lists
+import '../image_lists_3p_override.gcl' as image_lists_3p_override
+
+_parse_louhi_tag = lambda tag: {
+  parts = split(tag, '/')
+  ret = {
+    distro = parts[3]
+    arch = parts[4]
+  }
+}.ret
+
+config build = common.ops_agent_test {
+  params {
+    local _tag = _parse_louhi_tag(params.environment._LOUHI_TAG_NAME)
+    local _distro_arch = _tag.distro + '_' + _tag.arch
+    platforms = platforms = third_party_image_lists.get(_distro_arch, image_lists.get(_distro_arch, 'invalid_distro')).release
```

Comment:
This line is messed up in a few ways:

*  `third_party_image_lists` -> `image_lists_3p_override`
*  `platforms = platforms =`

It could also use a comment that `image_lists_3p_override` is used first, with a fallback to `image_lists` if it's not there.